bower.json

```diff
@@ -25,7 +25,7 @@
     "test"
   ],
   "dependencies": {
-    "ember": ">= 2.7.3 <= 3.0.0"
+    "ember": "2.7.3"
```

Comment:
ahh interesting! i suppose I did this change before glimmer came to be a thing. 💯